crates/ty_python_semantic/src/types/function.rs

```diff
@@ -1410,6 +1410,10 @@ pub enum KnownFunction {
     /// `dataclasses.field`
     Field,
 
+    /// `functools.total_ordering`
+    #[strum(serialize = "total_ordering")]
```

Comment:
this doesn't seem necessary, I think strum's default is fine here. No tests fail with it removed:

```suggestion
```